coriolis/tests/osmorphing/test_windows.py

```diff
@@ -735,27 +735,32 @@ def test_compile_static_ip_conf_from_registry_no_static_ip(self):
                       interfaces_reg_path),
         ])
 
-    def test_check_ips_info(self):
+    def test__get_static_nics_info(self):
+        # detected static IPs
+        ips_info = [{"ip_address": "10.0.0.16"},
+                    {"ip_address": "fe80::728a:688:1a92:baec"}]
         nics_info = [
-            {'ip_addresses': [mock.sentinel.ip_address]}
-        ]
-        ips_info = [
-            {'ip_address': mock.sentinel.ip_address}
-        ]
-        self.morphing_tools._check_ips_info(nics_info, ips_info)
-
-    def test_check_ips_info_different_ips(self):
-        nics_info = [
-            {'ip_addresses': [
-                mock.sentinel.ip_address, mock.sentinel.ip_address2]}
-        ]
-        ips_info = [
-            {'ip_address': mock.sentinel.ip_address}
-        ]
+            # no IP addresses on NIC
+            {"ip_addresses": [], "mac_address": "00:50:56:92:91:42"},
```

Comment:
The values, "00:50:56:92:91:" , ""fe81::728a:688:1a92:baec" .. etc .. can be defined as constants and can be referenced here. As we are using them multiple times.